lib/migration-payloads/validation/schema-validation.js

```diff
@@ -128,6 +132,28 @@ const validateContentType = function (payload) {
   });
 };
 
+const compactValidationsErrors = function (error) {
```

Comment:
nit: maybe something like `cleanErrors` or `cleanJoiNoiseFromErrors` would be more descriptive?